routes/service/[service]/component/[component]/[operationType]/op/[operationName]/schema.tsx

```diff
@@ -1,6 +1,5 @@
-import { getSchema } from "../../../../../../../../lib/fetch.ts";
-import { getAudienceFromParams } from "../../../../../../../../lib/utils.ts";
-import { parseData } from "../../../../../../../../islands/peek.tsx";
+import { getSchema } from "root/lib/fetch.ts";
```

Comment:
You're IDE may not know it...but there is a root alias to make these relative paths a bit more comprehensible.